data/fro/Zink_1984_Cleriadus/stemma.gv

```diff
@@ -0,0 +1,34 @@
+digraph {
+
+O -> x; 
+O -> y;
+
+y -> P3;
+y -> L;
+
+y -> T [style="dashed"];
+x -> T [style="dashed"];
+
+x -> A; 
+x -> V; 
+x -> xprime;
+xprime [label="x'"]
+
+xprime -> P1;
+xprime -> P2;
+
+V -> xprimeprime;
+xprimeprime [label="x''"]
+
+xprimeprime -> B; 
+xprimeprime -> C;
+
+B -> Vér [style="dashed"];
+C ->Vér [style="dashed"];
+
+Vér -> Len1; 
```

Comment:
Am I right in assuming that all these are printed editions ?

Reply:
Yes, they are prints.